src/equations/acoustic_perturbation_2d.jl

```diff
@@ -416,7 +416,8 @@ end
   c_mean_ll = u_ll[6]
   c_mean_rr = u_rr[6]
 
-  λ_max = (max(abs(v_ll), abs(v_rr)) + max(c_mean_ll, c_mean_rr)) * norm(normal_direction)
+  # The v_normals are already scaled by the norm
+  λ_max = max(abs(v_ll), abs(v_rr)) + max(c_mean_ll, c_mean_rr) * norm(normal_direction)
```

Comment:
In general this looks OK to me, but can you show/tell me where the velocities are scaled by the norm? Or is this happening implicitly in lines 408ff above?

Reply:
Yes, it is happening above because the `normal_direction` is not normalized one can think of it as
```
normal_direction = ( normal_vector ) * norm(normal_direction)
```

Reply:
Thanks!